models/models.js

```diff
@@ -1,16 +1,28 @@
 const db = require('../db/connection');
 
+//3
 exports.getTopicInfo = () =>{
     return db.query('SELECT * FROM topics;')
     .then(({rows}) => {
         return (rows);
     });
 };
 
+//4
 exports.getArticleInfo = () => {
     const sqlquery = 'SELECT articles.author,articles.title,articles.article_id,articles.topic,articles.created_at,articles.votes,COUNT(comments.article_id) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC;'
     return db.query(sqlquery)
     .then(({rows}) => {
         return (rows);
     });
-}
+};
+
+//5
+exports.getArticleIdInfo = (article_id) => {
+    return db.query('SELECT * FROM articles WHERE article_id=$1;',[article_id])
+    .then(({rows}) => {
+        return (rows)
```

Comment:
When we are sending back a single article, we don't want to send the array, just the single article object. In your test you are testing for an object so I'm not sure how this is currently passing?